recipe/meta.yaml

```diff
@@ -1,5 +1,6 @@
-{% set version = "2.1.4" %}
-{% set build_number = 1 %}
+{% set version = "3.0.0" %}
+{% set build_number = 0 %}
+{% set python_min = "3.10" %}
```

Comment:
XGBoost's Python minimum was bumped to 3.10 in PR: https://github.com/dmlc/xgboost/pull/10434

This change was included in XGBoost 3.0.0